packages/core/src/components/ozon-content/nodes/inhoud.node.tsx

```diff
@@ -7,6 +7,14 @@ export class OzonContentInhoudNode implements OzonContentNode {
   name = ["Inhoud", "ContainerBlocksType", "BlockMixedcontentMetMaximaleInlinesMarkersPopupsType"];
 
   render(node: Element, { mapNodeToJsx }: OzonContentNodeContext) {
-    return <div class="dso-rich-content">{mapNodeToJsx(node.childNodes)}</div>;
+    const editAction = node.getAttribute("wijzigactie");
+
+    const richContent = <div class="dso-rich-content">{mapNodeToJsx(node.childNodes)}</div>;
+
+    return (
+      (!editAction && richContent) ||
```

Comment:
Ok, nu zie ik beter wat er gebeurd 😄 

Maar eigenlijk zijn dit soort constructies vreselijk. Zullen we dit uitschrijven met een paar `if/if else` constructies?